drivers/drivers.go

```diff
@@ -0,0 +1,98 @@
+// Package drivers abstracts different object storages, such as local, s3, ipfs
+package drivers
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/net"
+)
+
+// Storages configured at start (used by broadcaster)
+var Storages []OSDriver
+
+// ExternalStorage own external storage, if configured
+var ExternalStorage OSDriver
+
+// LocalStorage single instance of local storage
+var LocalStorage *localOS
+
+// OSDriver common interface for Object Storage
+type OSDriver interface {
+	SaveData(jobID int64, streamID, name string, data []byte) (*net.TypedURI, error)
+	// IsExternal
+	IsExternal() bool
+	// GetInfo return info about object storage need to be able to write this storage
+	GetInfo() *net.OSInfo
+	// EvictManifestData removes memory cache for stream
+	EvictManifestData(mid string)
+}
+
+// AddStorageInstance adds storage driver instance to the list of storages used by broadcaster
+func AddStorageInstance(driver OSDriver) {
+	Storages = append(Storages, driver)
+}
+
+// NewDriver returns new driver
+func NewDriver(storageType net.OSInfo_Storage, info *net.OSInfo) OSDriver {
```

Comment:
net.OSInfo contains the storageType, so we can pass in just one parameter here, eg `func NewDriver(info *net.OSInfo) OSDriver`

Reply:
yeah, it was initially done for convenience, than I was calling this function manually with specific driver, now it can be removed.